pytests/repo/setup-repo.sh

```diff
@@ -28,34 +28,47 @@ mkdir -p ${BUILD_PATH}/BUILD \
 	 ${PUBLISH_PATH} \
 	 ${TEST_REPO_DIR}/yum.repos.d
 
-rpmbuild  --define "_topdir ${BUILD_PATH}" \
-	  -r ${BUILD_PATH} -ba ${REPO_SRC_DIR}/*.spec > /dev/null 2>&1
-
-cp -r ${BUILD_PATH}/RPMS ${PUBLISH_PATH}
-
-createrepo ${PUBLISH_PATH} > /dev/null 2>&1
-
-modifyrepo ${REPO_SRC_DIR}/updateinfo-1.xml ${PUBLISH_PATH}/repodata
-modifyrepo ${REPO_SRC_DIR}/updateinfo-2.xml ${PUBLISH_PATH}/repodata
-
 #gpgkey data for unattended key generation
-GPG_PASS=`openssl rand -base64 8`
+#GPG_PASS=`openssl rand -base64 8`
+GPG_PASS=1234
```

Comment:
Why is this done this way? If you need it in tests later, you should pass this down from config.py to this script and use the same in tests as `utils.gpg_pass`.

Reply:
This was left over, now removed. GPG_PASS isn't needed, keys are now generated without a passphrase. This has no bearing on the tests. But a passphrase does not work non-interactively with signing the packages.